etl/test/test_etl_load.py

```diff
@@ -99,32 +122,42 @@ async def test_etl_load_not_found_record(self, bq_client):
             },
         )
 
-    @run_as_sync
-    @patch('etl.load.main.bq.Client', autospec=True)
+    @patch('etl.load.main._get_bq_client', autospec=True)
+    @patch('etl.load.main.get_parser_instance')
     @patch('etl.load.main.call_parser')
-    async def test_etl_load_found_record_simple_payload(self, call_parser, bq_client):
+    def test_etl_load_found_record_simple_payload(
+        self, mock_call_parser, mock_get_parser_instance, mock_bq
+    ):
         """Test etl load simple payload"""
-        request = MagicMock(
-            args={}, spec=['__len__', 'toJSON', 'authorization', 'get_json']
-        )
-        request.get_json.return_value = {'request_id': '1234567890'}
-
-        query_row = MagicMock(args={}, spec=['body', 'type', 'submitting_user'])
-        query_row.body = ETL_SAMPLE_RECORD_2
-        query_row.type = '/bbv/v1'
-        query_row.submitting_user = 'user@mail.com'
 
-        query_job_result = MagicMock(args={}, spec=['__iter__', '__next__'])
-        query_job_result.total_rows = 1
-        query_job_result.__iter__.return_value = [query_row]
+        ETL_SAMPLE_RECORD_2 = json.dumps(
```

Comment:
Do we want to keep this as constant on top of the file, the same as RECORD_3?
Or if keeping in this test only then it should not be defined as constant (all upper case).
Just a bit of consistency I guess.

Reply:
I've moved the other as a definition into here, I want to discourage reuse here, I've stripped the payload down because it's not used.